pkg/reconciler/revision/revision.go

```diff
@@ -136,6 +138,15 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, rev *v1.Revision) pkgrec
 		return nil
 	}
 
+	// Spew is an expensive operation so guard the computation on the debug level
+	// being enabled.
+	// Some things, like PA reachability, etc are computed based on various labels/annotations
+	// of revision. So it is useful to provide this information for debugging.
+	log := logging.FromContext(ctx).Desugar()
```

Comment:
I'd prefer we didn't alias such a common package name with very similar method names.